content/using/install.md

```diff
@@ -91,43 +77,17 @@ http: live (insecure, loopback) on 12321
 ~sampel_marzod:dojo>
 ```
 
-When your ship is finished booting, you will see either the `~sampel_marzod:dojo>` or `~sampel_marzod:chat-cli/` prompt. If you see `:chat-cli`, press `Ctrl-X` to switch into Dojo.
+When your planet is finished booting, you will see either the `~sampel_marzod:dojo>` or `~sampel_marzod:chat-cli/` prompt. If you see `:chat-cli`, press `Ctrl-X` to switch into Dojo (the Urbit command line).
```

Comment:
This is wrong: it's not a planet. If you want to be more specific than ship, call it a comet.